src/components/pages/LandingPage.js

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from "react";
```

Comment:
Brought back the Landing Page created by Robbie. 
This one needs to be further styled later.